metricshub-engine/src/main/java/org/sentrysoftware/metricshub/engine/strategy/afterAll/AfterAllStrategy.java

```diff
@@ -0,0 +1,63 @@
+package org.sentrysoftware.metricshub.engine.strategy.afterAll;

```

Comment:
package names are in lower case. We should group all the surrounding strategies under `org.sentrysoftware.metricshub.engine.strategy.surrounding`.